### PR TITLE
Improve the bug report template

### PR DESCRIPTION
**Pull request by patrys**
New template for bug reports, default to applying the `triage` label.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by peelar — APPROVED** [review]
Looks good but I am wondering: is there a way we can test issue templates? And by test I mean render without merging?

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: environment
+    attributes:
+      label: "Environment"
+      description: Provide some information about your environment.
+      value: |
+        Saleor version: …
+        OS and version: …
```
[the remark this answers is not in the document]
Comment (patrys) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:48]: Feel free to update this, you can edit directly in GitHub.

**Review by maarcingebala — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ body:
     id: expected-behavior
     attributes:
       label: What did you expect to happen?
-      description: Tell us why you think the behavior described earlier is incorrect.
+      description: Describe why behavior described earlier is incorrect.
```
Comment (peelar) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:30]:
```suggestion
      description: Describe why the behavior described earlier is incorrect.
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ body:
     id: steps
     attributes:
       label: Steps to reproduce the problem
-      description: Tell us how to reproduce the problem starting with an empty database. You can use a free Saleor Cloud environment as a quick starting point.
+      description: Reproduction steps starting with an empty database. You can use a free Saleor Cloud environment as a quick starting point.
```
Comment (peelar) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:18]:
```suggestion
      description: Reproduction steps start with an empty database. You can use a free Saleor Cloud environment as a quick starting point.
```

Comment (patrys) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:18]: I don't think this is correct?

**Pull request merged**